### DISCUSSION TITLE
How do changes in survey design affect model structure?

### DISCUSSION THREAD

**Discussion by annepolyakov** [sdmTMB/sdmTMB#356] [General]
Hello! I am very excited to use the SDMtmb package for the data that I am working on! I am working with a trawl survey dataset spanning a number of decades from 1987-2022. One major change with the data is that from 1987-2007, survey locations were chosen randomly within each region, and each region was sampled every few years. From 2007-2022, 51 fixed locations were chosen across all regions and all of these locations were sampled consistently every year. Will it work to use this entire dataset to fit a spatiotemporal model? Or do I have to fit models to two separate datasets, one from 1987-2007, and one from 2008-2022? Thank you!

**Comment by Lewis-Barnett-NOAA** [comment]
Hi and thanks for the question.  This is the beauty of a model-based
estimator of this type, we don't have to make the assumptions a
design-based estimator would.  Thus, you can fit a combined model using the
two time periods while the spatial aspect of the model accounts for
differences in sample locations.  Of course, there may be differences that
arose between periods that the model can't disentangle, but you could
always try to give it a bit more information about the change (e.g., a
dummy variable indicating the sampling design identity perhaps?).

On Wed, Jul 17, 2024 at 10:33 PM annepolyakov ***@***.***>
wrote:

> Hello! I am very excited to use the SDMtmb package for the data that I am
> working on! I am working with a trawl survey dataset spanning a number of
> decades from 1987-2022. One major change with the data is that from
> 1987-2007, survey locations were chosen randomly within each region, and
> each region was sampled every few years. From 2007-2022, 51 fixed locations
> were chosen across all regions and all of these locations were sampled
> consistently every year. Will it work to use this entire dataset to fit a
> spatiotemporal model? Or do I have to fit models to two separate datasets,
> one from 1987-2007, and one from 2008-2022? Thank you!
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/pbs-assess/sdmTMB/discussions/356>, or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AAKMJP4HKIJDF5JQ6ISPM3DZM5HRFAVCNFSM6AAAAABLB5OL3SVHI2DSMVQWIX3LMV43ERDJONRXK43TNFXW4OZWHE2DQMBSGQ>
> .
> You are receiving this because you are subscribed to this thread.Message
> ID: ***@***.***>
>


-- 
Lewis Barnett, PhD (he/him)
Research Fish Biologist

NOAA Fisheries, Alaska Fisheries Science Center
7600 Sand Point Way NE, Bldg 4
Seattle, Washington 98115
Google Voice: (206) 526-4111

**Reply by annepolyakov** [reply]
Thank you so much for your quick and informative reply, that is great news! Yes, I think a dummy variable could work well, thank you!

**Comment by seananderson** [comment]
As Lewis says, for the most part, that is the type of problem a model-based approach can solve. However some caveats:
- If you have independent time slices (e.g., as.factor(year)) in your main effects then you won't be able to estimate a sampling design effect, assuming you don't have both sampling designs in one year. You _could_ do this, however, if your years were treated as being autoregressive in some way, say as a random walk (e.g., `time_varying = ~ 1, time = "year", time_varying_type = "rw0"`), but you are then imposing some degree of temporal smoothing on your index.
- In extreme sampling design cases (e.g., sampling a different region each year if there are strong spatial gradients in density), a model with independent years can sometimes mistake spatial variability for time variability. You can check this by creatively plotting your index with years coloured by regions sampled or similar and ensure there isn't an obvious pattern that does not make sense biologically. Additionally, you can check this with simulation, where you simulate a known dataset with properties roughly matching your system, impose your sampling design, and ensure you can recover the truth adequately and that this doesn't impose a pattern similar to what you see with the real data. I'm working on a project describing this more formally and showing possible solutions. If you're lucky, none of this applies to you!
